magicblock-chainlink/src/chainlink/config.rs

```diff
@@ -0,0 +1,66 @@
+use crate::remote_account_provider::config::RemoteAccountProviderConfig;
+
+#[derive(Debug, Clone, Default, PartialEq, Eq)]
+pub enum LifecycleMode {
```

Comment:
Are you absolutely sure we want to duplicate this enum here (and not just using the one in magicblock-config)?